media/css/firefox/whatsnew/whatsnew-56.scss

```diff
@@ -0,0 +1,388 @@
+            background-repeat: no-repeat;
+            content: '';
+            display: block;
+            height: 175px;
+            left: 50%;
+            margin-left: -150px;
+            position: absolute;
+            top: 0;
+            width: 300px;
+        }
+    }
+
+    .intro-content {
+        text-align: center;
+    }
+
+    &:after {
+        background: transparent url('/media/img/firefox/whatsnew_56/wave.svg') top center no-repeat;
+        bottom: -40px;
+        content: '';
+        display: block;
+        height: 355px;
+        left: 50%;
+        margin-left: -2122px;
+        position: absolute;
+        width: 3844px;
+    }
+
+    @media #{$mq-tablet} {
+        .intro-wrapper {
+            padding-top: 250px;
+
+            &:after {
+                height: 233px;
+                margin-left: -200px;
+                width: 400px;
+            }
+        }
+    }
+
+    @media #{$mq-desktop} {
+        .intro-wrapper {
+            padding-top: 0;
+
+            &:after {
+                height: 262px;
+                left: auto;
+                margin-left: 0;
+                right: 0;
+                width: 450px;
+            }
+        }
+
+        .intro-content {
+            text-align: left;
+            width: 45%;
+        }
+    }
+
+    @media #{$mq-desktop-wide} {
+        .intro-wrapper:after {
+            height: 319px;
+            width: 547px;
+        }
+    }
+}
+
+html[dir="rtl"] {
+    .intro {
+        .intro-wrapper:after {
+            margin-right: -150px;
+            right: 50%;
+        }
+
+        @media #{$mq-tablet} {
+            .intro-wrapper:after {
+                margin-right: -200px;
+            }
+        }
+
+        @media #{$mq-desktop} {
+            .intro-wrapper:after {
+                left: 0;
+                margin-right: 0;
+                right: auto;
+            }
+
+            .intro-content {
+                text-align: right;
+            }
+        }
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Features list
+
+.features-section {
+    @media #{$mq-tablet} {
+        padding-top: 40px;
+    }
+}
+
+.features-list {
+    padding-top: 240px;
+    position: relative;
+
+    li {
+        margin: 0 auto 40px;
+        max-width: 24em;
+        text-align: center;
+
+        &:first-child {
+            @include photon-horizontal-rule;
+            position: relative;
+        }
+    }
+
+    &:after {
+        @include at2x('/media/img/firefox/whatsnew_56/new-tab.png', 280px, 193px);
+        background-position: top right;
+        background-repeat: no-repeat;
+        content: '';
+        height: 203px;
+        margin-right: -140px;
+        position: absolute;
+        right: 50%;
+        top: 0;
+        width: 280px;
+    }
+
+    @media #{$mq-tablet} {
+        min-height: 435px;
+        padding-top: 0;
+        width: 50%;
+
+        li {
+            @include font-size-level3;
```

Comment:
I'm sure you're following design direction, but this copy looks really big in the page!

Reply:
Yeah agree, this is as per the design though (and it does look a bit too small at `font-size-level4`)